sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -248,36 +250,26 @@ def batch_event_handler(event):
         HTTP response
 
     """
-    command = event["detail"]["container"]["command"]
-
-    # Get params from batch job command
-    instrument = command[1]
-    data_level = command[3]
-    descriptor = command[5]
-    start_date = datetime.strptime(command[7], "%Y%m%d")
-    version = command[9]
-
     # Get job status
     job_status = (
         models.Status.SUCCEEDED
         if event["detail"]["status"] == "SUCCEEDED"
         else models.Status.FAILED
     )
 
-    status_params = {
-        "status": job_status,
-        "instrument": instrument,
-        "data_level": data_level,
-        "descriptor": descriptor,
-        "start_date": start_date,
-        "version": version,
-        "job_definition": event["detail"]["jobDefinition"],
-        "job_log_stream_id": event["detail"]["container"]["logStreamName"],
-        "container_image": event["detail"]["container"]["image"],
-        "container_command": " ".join(command),
-    }
-
-    update_status_table(status_params)
+    # We injected our table ID into the job name
+    job_id = event["detail"]["jobName"].split("-")[-1]
+
+    with db.Session() as session:
+        # Get the batch job by its ID
+        job = session.get(models.ProcessingJob, job_id)
+        # Make the updates
+        job.status = job_status
+        job.job_definition = event["detail"]["jobDefinition"]
+        job.job_log_stream_id = event["detail"]["container"]["logStreamName"]
+        job.container_image = event["detail"]["container"]["image"]
+        job.container_command = " ".join(event["detail"]["container"]["command"])
+        session.commit()
 
     return http_response(status_code=200, body="Success")
```

Comment:
I remember we had discussion about refactoring this `batch_event_handler` into it's own lambda. Should we save it for future PR?

Reply:
Yes, I would prefer to do it later because that involves updating the Lambda event listeners. Right now, this PR is completely separate from all AWS work.